src/main/java/graphql/schema/GraphQLObjectType.java

```diff
@@ -72,8 +72,7 @@ public GraphQLFieldDefinition getFieldDefinition(String name) {
 
     /**
      * @return This returns GraphQLInterface or GraphQLTypeReference instances, if the type
-     * references are not resolved yet. After they are resolved it contains only GraphQLInterface.
-     * Reference resolving happens when a full schema is built.
+     * references are not resolved yet. After they are resolved it contains only GraphQLInterface. * Reference resolving happens when a full schema is built.
      */
```

Comment:
JavaDoc weirdness here I think

Reply:
yes ... don't know how this happened, now fixed